contributors/devel/automation.md

```diff
@@ -6,9 +6,12 @@ Kubernetes uses a variety of automated tools in an attempt to relieve developers
 of repetitive, low brain power work. This document attempts to describe these
 processes.
 
-
 ## Submit Queue
 
+Submit Queue is deprecated. Kubernetes uses
```

Comment:
Should this be removed completely or be marked as deprecated?

Reply:
the submit queue doesn't even exist anymore (rm'ed) but people might have still heard of it.. I think we should probably keep this for a bit.

Reply:
I would change the heading to Tide, and this text can say something like "This project formerly used a Submit Queue, it has since been replaced by Tide"